### DISCUSSION TITLE
Maybe dumb question: why is email needed to download BlackHole off ExistentialAudio website?

### DISCUSSION THREAD

**Discussion by ghost** [ExistentialAudio/BlackHole#299] [Q&A]
Not really a problem; just wondering. It's just the first time I've seen a free open source project especially one hosted on github require a name and email for installer download. It says "subscribe for free download" what does subscribe mean here?

**Comment by devinroth** [comment]
You're free to use Homebrew to instal or build it yourself. I'm working to building a business and part of that is building my email list to notify users of updates and new releases.
